File: Solutions/1897. Redistribute Characters to Make All Strings Equal/1897. Redistribute Characters to Make All Strings Equal.go
func makeEqual(words []string) bool {
    cnt:=make([]int,26)
    for _,w:=range(words){
        for _,c:=range(w){
            cnt[c-'a']+=1
        }
    }
    for _,x:=range(cnt){
        if x%len(words)!=0{
            return false
        }
    }
    return true
}

File: Solutions/1897. Redistribute Characters to Make All Strings Equal/1897. Redistribute Characters to Make All Strings Equal.py
class Solution:
    def makeEqual(self, words: List[str]) -> bool:
        cnt=[0]*26
        for w in words:
            for c in w:
                cnt[ord(c)-ord('a')]+=1
        for x in cnt:
            if x%len(words)!=0:
                return False
        return True
                